yodeploy/hooks/tests/test_django.py

```diff
@@ -3,13 +3,14 @@
 from mock import PropertyMock, call, patch
 from pkg_resources import parse_version
 
-from yodeploy.hooks.django import DjangoApp
+from yodeploy.hooks.django import ApacheHostedDjangoApp, DjangoApp
 
 
 class TestDjangoHook(unittest.TestCase):
```

Comment:
Rename `TestDjangoHook` to `TestApacheHostedDjangoAppHook`?
I'm assuming the current `test_vhost_attributes ` would fail for `NginxHostedDjangoApp`.